tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -221,10 +221,22 @@
     <value>Configuration successfully applied</value>
     <comment>Message displayed when applying a configuration unit succeeds.</comment>
   </data>
-  <data name="ConfigurationUnitSummary" xml:space="preserve">
+  <data name="ConfigurationUnitSummaryFull" xml:space="preserve">
+    <value>{0} : {3} [{1} {2}]</value>
+    <comment>{Locked="{0}","{1}","{2}","{3}"}Summary text for a configuration unit. {0} is replaced by the configuration unit intent (e.g. Assert, Inform, Apply). {1} is replaced by the configuration unit name. {2} is replaced by the configuration unit id. {3} is replaced by the configuration unit description.</comment>
+  </data>
+  <data name="ConfigurationUnitSummaryMinimal" xml:space="preserve">
     <value>{0} : {1}</value>
     <comment>{Locked="{0}","{1}"}Summary text for a configuration unit. {0} is replaced by the configuration unit intent (e.g. Assert, Inform, Apply). {1} is replaced by the configuration unit name.</comment>
   </data>
+  <data name="ConfigurationUnitSummaryNoDescription" xml:space="preserve">
+    <value>{0} : {1} [{2}]</value>

```

Comment:
Do we need all of these variants to be localized? I'm a bit concerned that we may have more combinations we want to be able to handle and that's going to grow too much.

Reply:
I just followed the current style because I have no much context on why we are using a localized string for this, but folks, if you want I'm good with removing this strings.

Reply:
I don't remember getting guidance specifically about this. When I've added strings like these to localization it is because I'm not sure if other locales would use the same characters.
For example, I know for quotes some languages use 「these」 and some others use „these“. Not sure if that applies in this case, particularly considering the audience here is developers.

If we're keeping your current formatting, I think it's okay to localize these. IIRC my comment was thinking of adding the components to the string one by one always in the same order, instead of the current formatting where things change places depending on which ones are available.